druid/src/widget/button.rs

```diff
@@ -128,6 +129,13 @@ impl<T: Data> Widget<T> for Button<T> {
                 }
                 ctx.set_active(false);
             }
+            Event::MouseMove(_) => {
+                if !ctx.is_disabled() {
+                    ctx.set_cursor(&Cursor::Pointer);
+                } else {
+                    ctx.clear_cursor();
+                }
+            }
```

Comment:
I think this belongs in `lifecycle` instead of `event`. `Lifecycle::WidgetAdded` and `Lifecycle::DisabledChanged` are the events after which we need to update the cursor

Reply:
I'm not sure I understand. Modifying the cursor isn't a thing you can do from a lifecycle event and it doesn't make that much sense either as it needs to change based on whether you hover or not. No other widget seems to do it there either.

Reply:
You are right. I forgot, that `set_cursor` is only availible on `EventCtx`.
The problem is just that: If you click a button which then becomes disabled, the cursor only changes after the mouse moved again, you can see this in the disabled example.
But with the current implementation of cursor in `WidgetPod` and `Window` we probably can't change it :/